src/Sulu/Bundle/AdminBundle/Resources/js/components/Icon/Icon.js

```diff
@@ -14,8 +14,16 @@ export default class Icon extends React.PureComponent<Props> {
         const {className, name, onClick} = this.props;
         const iconClass = classNames(className, 'fa', 'fa-' + name);
 
+        const onClickProperties = onClick
+            ? {
+                onClick,
+                role: 'button',
+                tabIndex: 0,
+            }
+            : {};
+
         return (
-            <span className={iconClass} aria-hidden={true} onClick={onClick} />
```

Comment:
I had to remove the `aria-hidden` attribute, because if it is set, the `span` doesn't seem to appear to screen readers at all...